Messenger/Messenger.Core/Services/TeamService.cs

```diff
@@ -458,7 +459,7 @@ INSERT INTO
         /// <param name="role">The name of the role to remove</param>
         /// <param name="teamId">The id of the team to remove the role from</param>
         /// <returns>True if successful, false otherwise</returns>
-        public static async Task<bool> RemoveRole(string role, uint teamId)
+        public static async Task<uint> RemoveRole(string role, uint teamId)
```

Comment:
Make return type nullable to be consistent with the rest of the codebase.

Reply:
resolved in `e58447f`